SeleniumBasicsTest/tests/test_app_task2.py

```diff
@@ -0,0 +1,35 @@
+from appSource.app_tecklist import APPTechList
+import unittest
+
+
+class TestAPPTechListTask1(unittest.TestCase):
+
+    def setUp(self) -> None:
+        self.app = APPTechList()
+        print(self._testMethodName)
+        self.response = self.app.launch()
+
+    def test1_Launch_browser(self):

```

Comment:
already lunched application in setup method. please gave proper test case name

Reply:
i will change as per the error and i will PR
thanks very much